### DISCUSSION TITLE
Recent pushes alert now display on branches instead of main only

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#111301] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/10906554/6815f74c-c306-4dff-84a7-b2500e930121)

IDK if it was intended change, but it is not useful cuz the last push already clearly displays above fold to user so is redundant (also grammatically strange to refer to branch by name instead of "this branch")

**Comment by lin20260407** [comment]
Hi brother:

If GitHub has changed its behavior to show recent push alerts on branches other than just the main branch, and this is affecting your workflow, here are some steps and considerations on how to address this situation:

1. Adjust Notification Settings
First, check if GitHub has introduced new settings to control notifications for recent pushes. Sometimes, new features come with customizable settings that allow you to adjust how and where you receive alerts:

Visit your repository's settings on GitHub.
Look for options related to notifications, branches, or activity alerts.
Adjust the settings to fit your preference, if available.
2. Use GitHub Actions or Webhooks
If the direct option to control this behavior is not available, consider using GitHub Actions or webhooks to manage notifications:

GitHub Actions: Create a custom action that filters push notifications and sends alerts based on your criteria (e.g., only for pushes to the main branch).
Webhooks: Configure webhooks to listen to push events and handle them accordingly in a custom backend service, where you can filter out notifications from non-main branches.

**Reply by adamlui** [reply]
Hey @weitsunglin did you even readmy OP. I am saying if a potential bug was actually intended design, it is a bad one and needs improving, not how do I hide it (and if it's a bug then a programmer still needs to hide it not the user)

**Comment by lin20260407** [comment]
I agree with you, it's true that some features have user interaction issues that developers don't see.

**Reply by adamlui** [reply]
@weitsunglin well it's not an interaction issue either, being redundant makes it an issue of taking up valuable screen space without providing added value, which interaction does not need to occur for this issue to exist

**Reply by adamlui** [reply]
@weitsunglin but I like what you're doing what gave you the idea?

**Reply by lin20260407** [reply]
The concept of cleaning away the stink? It's like code.

**Reply by adamlui** [reply]
@weitsunglin are you AI

**Reply by lin20260407** [reply]
nonono i am a real person lol
